pkg/local-gadget-manager/local-gadget-manager_test.go

```diff
@@ -593,7 +593,7 @@ func TestNetworkGraph(t *testing.T) {
 		t.Fatalf("Failed to start the tracer: %s", err)
 	}
 
-	runTestContainer(t, containerName, "docker.io/library/alpine", "wget https://1.1.1.1", "")
+	runTestContainer(t, containerName, "docker.io/library/nginx", "nginx & sleep 1; curl 127.0.0.1", "")
```

Comment:
It seems you can get rid of the sleep 1 and just run:
```
nginx ; curl 127.0.0.1
```
because the nginx process will daemonize and go in the background when it's ready.

Tested with:
```
$ docker run -ti --rm nginx sh -c 'nginx ; curl 127.0.0.1'
2022/11/08 15:46:46 [notice] 7#7: using the "epoll" event method
2022/11/08 15:46:46 [notice] 7#7: nginx/1.23.2
2022/11/08 15:46:46 [notice] 7#7: built by gcc 10.2.1 20210110 (Debian 10.2.1-6) 
2022/11/08 15:46:46 [notice] 7#7: OS: Linux 5.18.9-200.fc36.x86_64
2022/11/08 15:46:46 [notice] 7#7: getrlimit(RLIMIT_NOFILE): 1073741816:1073741816
2022/11/08 15:46:46 [notice] 8#8: start worker processes
2022/11/08 15:46:46 [notice] 8#8: start worker process 9
2022/11/08 15:46:46 [notice] 8#8: start worker process 10
2022/11/08 15:46:46 [notice] 8#8: start worker process 11
2022/11/08 15:46:46 [notice] 8#8: start worker process 13
<!DOCTYPE html>
<html>
<head>
<title>Welcome to nginx!</title>
<style>
html { color-scheme: light dark; }
body { width: 35em; margin: 0 auto;
font-family: Tahoma, Verdana, Arial, sans-serif; }
</style>
</head>
<body>
<h1>Welcome to nginx!</h1>
<p>If you see this page, the nginx web server is successfully installed and
working. Further configuration is required.</p>

<p>For online documentation and support please refer to
<a href="http://nginx.org/">nginx.org</a>.<br/>
Commercial support is available at
<a href="http://nginx.com/">nginx.com</a>.</p>

<p><em>Thank you for using nginx.</em></p>
</body>
</html>
127.0.0.1 - - [08/Nov/2022:15:46:46 +0000] "GET / HTTP/1.1" 200 615 "-" "curl/7.74.0" "-"
```